packages/vsx-registry/src/browser/vsx-extensions-model.ts

```diff
@@ -139,12 +139,17 @@ export class VSXExtensionsModel {
             const searchResult = new Set<string>();
             for (const data of result.extensions) {
                 const id = data.namespace.toLowerCase() + '.' + data.name.toLowerCase();
+                const extension = await this.api.getLatestCompatibleExtension(id);
```

Comment:
why is it async? we discussed that search should be fast with only one remote call

Reply:
Currently it is `async` since getting the api version from the environment is `async`. 
I will need to modify the code to fetch and cache this variable `onStart` so I can then retrieve it `sync`.